src/main/resources/application-docker.yml

```diff
@@ -23,7 +23,7 @@ consentmanager:
   keycloak:
     baseUrl: http://keycloak:8080/auth
     clientId: consent-manager
-    clientSecret: ${CONSENTMANAGER_KEYCLOAK_CLIENTSECRET}
+    clientSecret: "0e3f27a0-c167-45b4-a95b-ca1ba019b1bf"
```

Comment:
it should be as how it was, otherwise environment variables will get replaced at all.

Reply:
Everything else is good except this.

Reply:
Fixed